lib/ui/pages/news_list/primary_news_list_item.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:allthenews/ui/pages/news_list/primary_news_list_entity.dart';
+import 'package:flutter/material.dart';
+
+abstract class _Constants {
+  static const double newsImageRadius = 15;
+  static const double newsImageLayerOpacity = 0.7;
+  static const double authorImageRadius = 15;
+  static const double authorRightPadding = 8;
+  static const double stackItemPadding = 20;
+}
+
+class PrimaryNewsListItem extends StatelessWidget {
+  final PrimaryNewsListEntity news;
+
+  PrimaryNewsListItem({Key key, @required this.news})
+      : assert(news != null),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Stack(
+        children: <Widget>[
+          ClipRRect(
+            borderRadius: BorderRadius.circular(_Constants.newsImageRadius),
+            child: Container(
+              color: Colors.black,
+              child: Opacity(
+                opacity: _Constants.newsImageLayerOpacity,
```

Comment:
bardzo fajny patent, propsuję :D

Reply:
a dziękuję, jestem ciekawy jak to będzie wyglądać na tle całego widoku

Reply:
bo na designach też są przyciemnione chyba te obrazki, żeby tekst biały było widać